src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -2054,6 +2054,12 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
 
             var textBlock = values[0] as TextBlock;
             var viewModel = values[1] as SearchViewModel;
+
+            // In some cases viewModel can be null. Mostly it's because workspace has not loaded yet.
+            // But converter has been already called.
+            if (viewModel == null)
```

Comment:
@aosyatnik This is a very small thing, but I believe our coding style is to use brackets for `if` statements, even if they're only one line long. ex:
```c#
if(viewModel == null)
{
  return new Thickness(0, 0, textBlock.ActualWidth, textBlock.ActualHeight);
}
```

Also, you've go an extra semicolon at the end there :)